source/api_documentation/transactions_api/_network_param_table.rst

```diff
@@ -223,7 +223,7 @@
 
   * - transaction_type
     - Type
-    - The type of transaction - Call, Reported Conversion or Signal.
+    - The type of transaction - Call, Post Call Event, Reported Conversion, or Signal.
```

Comment:
Is the outputted value `Post Call Event` (with spaces), even with format is API?  And the input type is with no spaces?  I think that will cause confusion / bugs too..

Reply:
Discussed in person, going to add a TODO to the TransactionAPI Controller to hopefully address this in the future.